tracer/src/Datadog.Trace/Agent/MessagePack/SpanMessagePackFormatter.cs

```diff
@@ -98,13 +113,13 @@ public int Serialize(ref byte[] bytes, int offset, in TraceChunkModel traceChunk
                 // or if its parent can also be found in the same chunk, so we use SpanModel
                 // to pass that information to the serializer
                 var spanModel = traceChunk.GetSpanModel(i);
-                offset += Serialize(ref bytes, offset, in spanModel);
+                offset += Serialize(ref bytes, offset, in spanModel, traceChunk.InAzureAppService, traceChunk.AzureAppServiceSettings);
```

Comment:
This is getting a little big now. I think what you have right now will definitely work, but ideally I'd like to put the AzureAppService info into the constructor of the SpanMessagePackFormatter since this shouldn't be changing at all for the entire time that the application is running. Then we could use private fields instead of propagating the information all over the place

Reply:
As it's a singleton, it would be making it harder to test. But we actually already have the data in `SpanModel`. I forgot that `SpanModel` contained the TtraceChunkModel`. Fixed in 00cd70fc4b7871c3a9846286093009db3706ddfd